### ISSUE TITLE
Relation to other more room acoustic relates Python packages

### ISSUE THREAD

**Issue by hagenw** [thejasvibr/bat_beamshapes#19]
Related to https://github.com/openjournals/joss-reviews/issues/3740

In the paper you state 

> To my knowledge there are no computational implementations of sound
> source directivities that are open-source and developed using modern software practices such
> as version-control and unit-testing

which seems to be correct to my knowledge.

But to foster more interaction with the non-bioacoustic community it would be great to add a few statements how it relates to other Python packages, that include some kind of directivity estimations, e.g. https://pyroomacoustics.readthedocs.io/en/pypi-release/pyroomacoustics.directivities.html or https://github.com/AppliedAcousticsChalmers/levitate (see https://github.com/AppliedAcousticsChalmers/levitate/blob/c6ad1038327abfc82a5402b42019b69e52f7f5c4/levitate/transducers.py#L720).

**Comment by thejasvibr** [comment]
Thanks for this pointer - have mentioned the Pyroomacoustics and Levitate packages in the manuscript now.

**Comment by hagenw** [comment]
Thanks for mentioning them. It would maybe help to add a few more words on comparing them. I also do not really know/have used the other packages. This means a user I would have no idea when I should go for `beamshapes` or `pyroomacoustics` when reading the paper.

**Comment by hagenw** [comment]
I would like to reopen this issue, as I don't think the current state in the paper is sufficient.

**Comment by faroit** [comment]
@thejasvibr would you be willing to give this another pass?

**Comment by thejasvibr** [comment]
hello @faroit @hagenw - will give another try to make this portion clearer. Apologies for the delay in response - was on leave till today.

**Comment by thejasvibr** [comment]
Have included clarification in the MS as of [a2b5840...](https://github.com/thejasvibr/bat_beamshapes/commit/a2b5840d863b2571ae29f742beea73873581a36e). 

To be specific, I've actually only focussed on the ```levitate``` package as I realised that ```pyroomacoustics```  actually implements the cardiod pattern - which describes a 'shape' rather than a directivity pattern linked to a specific sound source model. ```levitate``` implements relatively 'simple' sound source directivities with analytical solutions - while ```beamshapes``` stands out in implementing directivities that also don't have analytical solutions.
